.github/actions/get-cache/action.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Get Cache Keys'
+description: 'Action to get cache'
+inputs:
+  github-token:
+    description: 'GitHub token'
+    required: true
+  enable-cache:
+    description: 'Enable cache'
+    default: ''
+  keys:
+    description: 'Keys'
+    default: 'false'
+outputs:
+  keys:
+    description: 'Keys'
+    value: ${{ steps.prepare.outputs._CACHE_KEYS }}
+runs:
+  using: 'composite'
+  steps:
+    - name: Adding required env vars
+      uses: actions/github-script@v7
+      env:
+        github-token: ${{ inputs.GITHUB_TOKEN }}
+      with:
+        script: |
+          core.exportVariable('ACTIONS_CACHE_URL', 'https://cache.dev01.devland.is/')
+          core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env['ACTIONS_RUNTIME_TOKEN'])
+          core.exportVariable('ACTIONS_RUNTIME_URL', process.env['ACTIONS_RUNTIME_URL'])
+    - name: Get cache
+      id: prepare
+      shell: bash
+      env:
+        _CACHE_KEYS: ${{ inputs.keys }}
+        ENABLE_CACHE: ${{ inputs.enable-cache }}
```

Comment:
I have a hard time tracking this env throughout the stack. Where is it used?

Reply:
_CACHE_KEYS is a string of json with all hash keys, only used if this is not inited.

ENABLE_CACHE enables or disables cache steps.